packages/react-ui-core/src/List/List.js

```diff
@@ -49,10 +55,18 @@ export default class List extends PureComponent {
     return `${this.id}-${index}`
   }
 
-  renderListItem(listItem) {
+  itemMetaProps(listItem) {
     return parseArgs(listItem, DefaultListItem)
```

Comment:
what are itemMetaProps?

Reply:
This should probably be renamed since it's returning both a component and props.